apisix/admin/routes.lua

```diff
@@ -17,19 +17,13 @@
 local expr = require("resty.expr.v1")
 local core = require("apisix.core")
 local apisix_upstream = require("apisix.upstream")
+local resource = require("apisix.admin.resource")
 local schema_plugin = require("apisix.admin.plugins").check_schema
-local utils = require("apisix.admin.utils")
 local tostring = tostring
 local type = type
 local loadstring = loadstring
 
 
-local _M = {
-    version = 0.2,
-    need_v3_filter = true,
-}
-
-
 local function check_conf(id, conf, need_id)
```

Comment:
Look like we can also share the id check logic? It is the same in other resources.